### ISSUE TITLE
Is manifest-root supported for deps.edn?

### ISSUE THREAD

**Issue by vemv** [reifyhealth/lein-git-down#50]
Hi!

I have a config like this:

```
{eftest/eftest
  {:coordinates vemv/eftest, :manifest-root "eftest"}}
```

where deps.edn is not located at the root of the `vemv/eftest` repo, but within its `eftest` folder.

Is that supposed to work? I'm not sure if deps.edn counts as a "manifest file" which makes me hesitate.

Cheers - V

**Comment by manderson202** [comment]
Hi, yes, this should be supported for `deps.edn` as well. Are you encountering any issues with this set up?

**Comment by vemv** [comment]
Yes, the generated artifact I can find in ~/.m2 lacks any sources, I only see a few meta files

**Comment by manderson202** [comment]
Ok, thanks for the report, will leave this open to track

**Comment by vemv** [comment]
Here's the repo in question (wasn't public before) https://github.com/vemv/eftest
